### DISCUSSION TITLE
Loop Accordion Widget

### DISCUSSION THREAD

**Discussion by steveindzine** [elementor/elementor#27822] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I have copied this from https://github.com/orgs/elementor/discussions/23240 as it was closed prematurely:

Allow building an accordion with content from a query.

Use case: FAQs as CPT could be dynamically queried and displayed in accordion format without having to statically add them to an accordion widget in each part of the site where they are to be displayed. This could be combined with supported third-party facet/filter/search plugins to filter them out like any other query loop.

### Describe the solution you'd like.

Simply replicate the existing accordion widget with the ability to specify a query as the data source for the accordion items rather than requiring the content to be entered in the widget settings itself. (The way the Loop Carousel widget is to the Carousel widget, pretty much!)

### Describe alternatives you've considered.

Nested accordion was recommended when this was originally requested: https://github.com/orgs/elementor/discussions/23240

But it doesn't actually do what is being asked. Adding an accordion to a loop just creates lots of independant accordions, we need one accordion with the content pulled from the CPT. The loop itself would need a widget which is one element of an accordion, which is pulled into the accordion loop itself. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by hxinen** [comment]
I second this, almost 2026 and still no way of handling this?
